advanced_functionality/scikit_learn_bring_your_own_LightGBM_model/scikit_learn_bring_your_own_LightGBM_model.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    data = fetch_california_housing()
</pre><p>Is it possible to use the hosted version&nbsp;<code>s3://sagemaker-sample-files/datasets/tabular/california_housing/cal_housing.tgz</code>?</p><p>See this notebook for similar use:</p><p><a href="https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-pipelines/tabular/train-register-deploy-pipeline-model/train%20register%20and%20deploy%20a%20pipeline%20model.ipynb" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-pipelines/tabular/train-register-deploy-pipeline-model/train%20register%20and%20deploy%20a%20pipeline%20model.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Using the hosted version would needlessly complicate the data step. Could it please be left as is?</p>